vendor/bat-native-ledger/src/bat/ledger/internal/legacy/publisher_settings_properties.cc

```diff
@@ -48,4 +72,147 @@ bool PublisherSettingsProperties::operator!=(
   return !(*this == rhs);
 }
 
+base::Value::Dict PublisherSettingsProperties::ToValue() const {
+  base::Value::Dict dict;
+
+  dict.Set(kMinPageTimeBeforeLoggingAVisitKey,
+           base::Int64ToValue(min_page_time_before_logging_a_visit));
+  dict.Set(kMinVisitsForPublisherRelevancy,
+           base::Int64ToValue(min_visits_for_publisher_relevancy));
+  dict.Set(kAllowNonVerifiedSitesInListKey, allow_non_verified_sites_in_list);
+  dict.Set(kAllowContributionToVideosKey, allow_contribution_to_videos);
+
+  base::Value::List monthly_balances_list;
+  for (const auto& monthly_balance : monthly_balances) {
```

Comment:
```
for (const auto& [key, report_balance] : monthly_balances) { /* ... */ }
```

Reply:
Fixed in 98689b867f823358b8cdfafa61bdef53370f3845.